src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/resource/RecordARecallController.kt

```diff
@@ -21,7 +21,7 @@ class RecordARecallController(
   private val recordARecallService: RecordARecallService,
 ) {
   @PostMapping(value = ["/record-a-recall/{prisonerId}"])
```

Comment:
this endpoint url would prob be better as 
`/record-a-recall/calculate/{prisonerId}`

but will address that separately